### PR TITLE
Add 2019 dates

### PR DESCRIPTION
**Pull request by chao-xian**
Adds bank holidays and clock changes for 2019.

### CODE REVIEW

**Comment by chao-xian** [comment]
[Add bank holidays](https://trello.com/c/cMIvHbYd/767-add-bank-holidays)

File: lib/data/bank-holidays.json [hunk 1]
```diff
@@ -362,6 +362,56 @@
                 "bunting": true
               }
             ],
+            "2019": [
+                {
+                    "title": "bank_holidays.new_year",
+                    "date": "01/01/2019",
+                    "notes": "",
+                    "bunting" :true
```
Comment (fofr) [inline lib/data/bank-holidays.json]: Weird whitespace here.

**Comment by fofr** [comment]
Code looks good. Not sure if you need sign off on the dates.

**Comment by chao-xian** [comment]
Thanks @fofr. Yeah the Content Team will check it but I was hoping that the Heroku Review apps would be available. Have given them screenshots for now.

File: lib/data/bank-holidays.json (hunk 1 above)
Comment (chao-xian) [reply lib/data/bank-holidays.json]: Fixed in https://github.com/alphagov/calendars/pull/152/commits/5068a290d0df375bbe6bea87b499326134dabdea

Comment (fofr) [reply lib/data/bank-holidays.json]: Still seeing varieties of:
```
"bunting" :true
"bunting":true
"bunting": true
```

Should all match the last one?

Comment (chao-xian) [reply lib/data/bank-holidays.json]: 😓  fixed -https://github.com/alphagov/calendars/pull/152/commits/d435987e3c39f9b0792181397c4f9086483833c6

**Pull request merged**